tools/common/Assembly.cs

```diff
@@ -236,6 +244,14 @@ public void ExtractNativeLinkInfo ()
 					if (!Application.IsUptodate (FullPath, path)) {
 						Application.ExtractResource (assembly.MainModule, libraryName, path, false);
 						Driver.Log (3, "Extracted third-party binding '{0}' from '{1}' to '{2}'", libraryName, FullPath, path);
+						Driver.Log (3, "    ForceLoad: {0}", linkWith.ForceLoad);
+						Driver.Log (3, "    Frameworks: {0}", linkWith.Frameworks);
+						Driver.Log (3, "    IsCxx: {0}", linkWith.IsCxx);
+						Driver.Log (3, "    LinkerFlags: {0}", linkWith.LinkerFlags);
+						Driver.Log (3, "    LinkTarget: {0}", linkWith.LinkTarget);
+						Driver.Log (3, "    NeedsGccExceptionHandling: {0}", linkWith.NeedsGccExceptionHandling);
+						Driver.Log (3, "    SmartLink: {0}", linkWith.SmartLink);
+						Driver.Log (3, "    WeakFrameworks: {0}", linkWith.WeakFrameworks);
```

Comment:
Good idea :) but could you avoid the duplicate code ? that will likely drift over time